modules/ossm-kiali-architecture.adoc

```diff
@@ -0,0 +1,25 @@
+////
+This CONCEPT module included in the following assemblies:
+-ossm-kiali.adoc
+////
+
+[id="ossm-kiali-architecture_{context}"]
+= Kiali architecture
+
+Kiali is composed of two components: the Kiali application and the Kiali console.
+
+* *Kiali application* (back end) – This component runs in the container application platform and communicates with the service mesh components, retrieves and processes data, and exposes this data to the console. The Kiali application does not need storage. When deploying the application to a cluster, configurations are set in ConfigMaps and secrets.
+
+* *Kiali console* (front end) – The Kiali console is a web application. 	The Kiali application serves the Kiali console, which then queries the application for data in order to present it to the user.
+
+In addition, Kiali depends on external services and components provided by the container application platform and Istio.
+
+* *Red Hat Service Mesh* (Istio) - Istio is a Kiali requirement. Istio is the component that provides and controls the service mesh. Although Kiali and Istio can be installed separately, Kiali depends on Istio and will not work if it is not present. Kiali needs to retrieve Istio data and configurations, which are exposed through Prometheus and the cluster API.
+
+* *Prometheus* - A dedicated Prometheus instance is included as part of the {ProductName} installation.  When Istio telemetry is enabled, metrics data is stored in Prometheus. Kiali uses this Prometheus data to determine the mesh topology, display metrics, calculate health, show possible problems, and so on. Kiali communicates directly with Prometheus and assumes the data schema used by Istio Telemetery. Prometheus is an Istio dependency and a hard dependency for Kiali, and many of Kiali’s features will not work without Prometheus.
```

Comment:
is the "ProductName" here equivalent to the "product-title" on line 21?

Reply:
ProductName = Service Mesh 
Product-Title = OpenShift